selfdrive/car/tesla/radar_interface.py

```diff
@@ -159,6 +163,7 @@ def _update(self, updated_messages):
     errors = []
     if not self.rcp.can_valid:
       errors.append("canError")
+      self.tinklaClient.logCANErrorEvent(source="radar_interface", canMessage=0, additionalInformation="Invalid CAN Count")
```

Comment:
how can we extract the message to log from the can parser (rcp)? Or where would it make sense for this to be called from?